squirrel.frontier/src/main/java/org/dice_research/squirrel/queue/domainbased/MongoDBDomainBasedQueue.java

```diff
@@ -12,6 +12,7 @@
 import org.dice_research.squirrel.data.uri.serialize.Serializer;
 import org.dice_research.squirrel.data.uri.serialize.java.SnappyJavaUriSerializer;
 import org.dice_research.squirrel.queue.AbstractDomainBasedQueue;
+import org.rdfhdt.hdt.util.Histogram;
```

Comment:
Why is this class imported?

Reply:
this should have come in one of the merges, removed